### PR TITLE
Skip  t_ili2db_meta_attrs if it does not exist 

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

File: projectgenerator/libqgsprojectgen/dbconnector/pg_connector.py [hunk 1]
```diff
@@ -142,6 +157,9 @@ def get_tables_info(self):
         return []
 
     def get_meta_attrs(self, ili_name):
+        if not self._table_exists('t_ili2db_meta_attrs'):
```
Comment (signedav) [inline projectgenerator/libqgsprojectgen/dbconnector/pg_connector.py]: You could use `PG_METAATTRS_TABLE` here.

File: projectgenerator/tests/test_dump.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+start_app()
+
+
+class TestCustomDump(unittest.TestCase):
+
+    def test_ili2pg_dump_without_metattr(self):
+        myenv = os.environ.copy()
+        myenv['PGPASSWORD'] = 'docker'
+        call(["pg_restore", "-Fc", "-hpostgres", "-Udocker", "-dgis", testdata_path("dumps/_nupla_dump")], env=myenv)
```
Comment (signedav) [inline projectgenerator/tests/test_dump.py]: I always make spaces between parameters, like `-h postgres` but I guess this works as well.

Comment (m-kuhn) [reply projectgenerator/tests/test_dump.py]: With `pg_restore` also? I normally also make those spaces, but had trouble here.

Comment (signedav) [reply projectgenerator/tests/test_dump.py]: Also on postgresql.org/docs there is the example for `pg_restore` with spaces. I don't know where the problems come from, but I suppose when it works here without them, then it's fine as well.

Comment (m-kuhn) [reply projectgenerator/tests/test_dump.py]: I also had to use an environment variable for the password, might be related to an old version of pg_restore. I'll keep this working approach and merge.

Thanks for the review

**Pull request merged**